nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Cb fix order and customer controllers

Fixes:
-added ability to include payments/products on customer
-added ability to include customers on orders

Changes proposed in this request & reasons behind organizational or architectural decisions:
-N/A

Steps to test:
-Run most recent dummy data sql
-start API
-enter the following URLs:
-http://localhost:5000/Customer
http://localhost:5000/Customer?_include=noOrders
http://localhost:5000/Customer?_include=products
http://localhost:5000/Customer?_include=payments
http://localhost:5000/Customer/1
http://localhost:5000/Customer/1_include=products
http://localhost:5000/Customer/1_include=payments

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-N/A

Commit message:
-fixed customer controller and tests

Link to feature ticket:
-https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/5
-https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/2

@ambitions-albatrosses
